Dev/src/study_tools/flashcards.py

```
@@ -0,0 +1,39 @@
+"""Generate Anki deck from summaries."""
+
+import uuid
+from pathlib import Path
+
+# heavy imports in main()
+
+from .utils import load_config
+
+
+def main():
```

Pylint (3.3.7)

[refactor] Too many local variables (19/15)

(R0914)